modules/sdk/gradle-plugins-poshi-runner/src/main/java/com/liferay/gradle/plugins/poshi/runner/PoshiRunnerPlugin.java

```diff
@@ -153,6 +154,18 @@ public void execute(Configuration configuration) {
 	protected void addDependenciesPoshiRunner(
 		Project project, PoshiRunnerExtension poshiRunnerExtension) {
 
+		File poshiPropertiesFile =
+			poshiRunnerExtension.getPoshiPropertiesFile();
+
+		if (poshiPropertiesFile.exists()) {
+			File tmpPoshiPropertiesFile = new File(
```

Comment:
You're not copying the file here. Have a look [here](https://docs.oracle.com/javase/7/docs/api/java/nio/file/Files.html#copy%28java.nio.file.Path,%20java.nio.file.Path,%20java.nio.file.CopyOption...%29).